deploy/manifests/walrus-file.yaml

```diff
@@ -0,0 +1,52 @@
+resources:
+  - name: walrus-db
+    type: postgresql
+    attributes:
+      username: root
+      database: walrus
+      engine_version: "14"
+  - name: walrus-redis
+    type: redis
+    attributes: {}
+  - name: walrus-casdoor
+    type: containerservice
+    attributes:
+      containers:
+        - image: sealio/casdoor:v1.344.0-seal.1
+          ports:
+            - internal: 8000
+              external: 8000
+          execute:
+            privileged: true
```

Comment:
Do we need this?

Reply:
Referred to docker-compose.yaml, but removing it seems no difference.

Reply:
Let's also remove it in compose if it's useless

Reply:
removed